azurerm/resource_arm_application_gateway_test.cer

```diff
@@ -0,0 +1,21 @@
+-----BEGIN CERTIFICATE-----

```

Comment:
since the gateway requires .pfx format, can't use the tls_locally_signed_cert since it requires .pem format